src/module/home.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable */
+import { getMovie, getLikes, getComments } from './Api.js';
+const section = document.querySelector('.container');
+
+const homePage = async () => {
+  const data = await getMovie();
+  const likesData = await getLikes();
+  const comments = await getComments();
+  console.log(likesData);
+  data.forEach((movie) => {
+    const likes = likesData.find((like) => like.item_id === movie.likes) || [];
+    const like = likes.likes || 0;
+    section.innerHTML += `
+          <div class="item"> 
+          <div class = "img-container">
+             <img src="${movie.image.medium}" alt="">
+         </div>
+         <div class =" item-content">
+         <div class='texts-container'>
+         <p class="movie-title">${movie.name}</p>
+         <button class="btn-likes"><img class = "like-heart" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ2K8owv-3mxT9d_LUeJ4KPt4car7cer1v_EttngB0fqED1mviLqRKyNfq3KjFvfpsK6Fw&usqp=CAU" alt=""><span>${like}likes</span></button>
+         </div>
+          <button id="${APP_ID}" class="button">Comment</button>
+          <div>
+      </div>
+          `;
+  });
+  document.querySelector(
+    '#main-title'
+  ).textContent = `Top ${data.length} Movies`;
+
+  const commentBtn = document.querySelectorAll('.button');
+  commentBtn.forEach((btn) => {

```

Comment:
### Hi @MxoDevops , 
## Good job so far!
To Highlight! 👏 🟢
✔️ Clean/organized code.


There are some issues that you still need to work on to get this project completed, but you are almost there!
![Alt Text](https://i.pinimg.com/originals/10/94/a3/1094a3e620ebea4cd6cf2d0aceaa8f52.gif)

# Status: Required Changes ♻️

- [x] Please open different files for comment popup so that  we can find bugs easily
- [ ] Please kindly work on the ApI its not fetching  .
- [ ] fix linter errors.